components/style/properties.mako.rs

```diff
@@ -4783,6 +4852,92 @@ pub mod longhands {
         pub use properties::longhands::transition_duration::{get_initial_single_value};
         pub use properties::longhands::transition_duration::{get_initial_value, parse, parse_one};
     </%self:longhand>
+
+    // CSS Flexible Box Layout Module Level 1
+    // http://www.w3.org/TR/css3-flexbox/
+
+    ${new_style_struct("Flex", is_inherited=False)}
+
+    // Flex container properties
+    ${single_keyword("flex-direction", "row row-reverse column column-reverse")}
+    ${single_keyword("flex-wrap", "nowrap wrap wrap-reverse")}
+    ${single_keyword("justify-content", "flex-start flex-end center space-between space-around")}
+    ${single_keyword("align-items", "stretch flex-start flex-end center baseline")}
+    ${single_keyword("align-self", "auto flex-start flex-end center baseline stretch")}
+    ${single_keyword("align-content", "stretch flex-start flex-end center space-between space-around")}
+
+    // Flex item properties
+    ${integer_type_or_auto("order", "i32", "Specified(0)")}
+    ${integer_type_or_auto("flex-grow", "u32", "Specified(0)", "value >= 0")}
+    ${integer_type_or_auto("flex-shrink", "u32", "Specified(1)", "value >= 0")}
```

Comment:
Yes, please remove the parsing of unsupported properties. Especially since they are one-line macro calls, they’re easy to add back when support is added.